app/decorators/application_decorator.rb

```diff
@@ -1,5 +1,33 @@
+# frozen_string_literal: true
+
 class ApplicationDecorator < Draper::Decorator
   delegate_all
-  
+
   self.include_root_in_json = true
+
+  def api_selector_api_link
+    raise NotImplementedError
```

Comment:
`NotImplementedError` is not correct, `NoMethodError` is the right one
```suggestion
    raise NoMethodError, __method__
```